src/Nest/CommonAbstractions/Infer/Field/Field.cs

```diff
@@ -4,54 +4,82 @@
 using System.Linq.Expressions;
 using System.Reflection;
 using Elasticsearch.Net;
+using Newtonsoft.Json;
 
 namespace Nest
 {
-	[ContractJsonConverter(typeof(FieldJsonConverter))]
+	/// <summary>
+	/// A field within Elasticsearch
+	/// </summary>
+	[JsonConverter(typeof(FieldJsonConverter))]
 	[DebuggerDisplay("{DebugDisplay,nq}")]
 	public class Field : IEquatable<Field>, IUrlParameter
 	{
 		private readonly object _comparisonValue;
 		private readonly Type _type;
 
-		public Field(string name, double? boost = null)
+		public Field(string name, double? boost = null, string format = null)
```

Comment:
Isn't this an undocumented breaking change? Just had a MissingMethodException in one of my projects. See https://stackoverflow.com/a/23517490/1282778

Reply:
It absolutely is! This one is totally on me for not running: https://github.com/nullean/AssemblyDiffer before doing the last release as we normally would do. Will see if we can rectify this in the next release. Thank you for bringing this to our attention @filpen